stable/heapster/values.yaml

```diff
@@ -6,9 +6,10 @@ image:
   tag: v1.3.0
   pullPolicy: IfNotPresent
 ## Here labels can be added to the heapster deployment
-# labels:
-#   kubernetes.io/cluster-service: "true"
-#   kubernetes.io/name: "Heapster"
+## labels: {
+##   "kubernetes.io/cluster-service": "\"true\"",
+##   "kubernetes.io/name": "\"Heapster\""
+## }
 labels: {}
```

Comment:
The old way should work. You shouldn't need to turn it into an object rather than yaml itself. Do you have an issue with more details?

You can see how others, such as [traefik](https://github.com/kubernetes/charts/blob/88804d2c9de2eedaff82c99f2797d3ecde3c0c0c/stable/traefik/values.yaml#L67) have done this.

Reply:
Yeah I tried using the following format: 
  labels:
    key: value
 However I get errors such as:
Error: release fuzzy-chipmunk failed: Service in version "v1" cannot be handled as a Service: [pos 168]: json: expect char '"' but got char 't'
I had not created an issue yet because I didn't realize it should be an issue.
I will go create an issue though and add more details in the issue.
Is there anything in particular that you can think of that would help?

Reply:
Did you try turning:
```
# labels:
#   kubernetes.io/cluster-service: "true"
#   kubernetes.io/name: "Heapster"
 labels: {}
```

into:
```
labels:
  kubernetes.io/cluster-service: "true"
  kubernetes.io/name: "Heapster"
```

Notice that `labels` is only listed as a top level thing once.